gui/mozregui/addons_editor.py

```diff
@@ -17,7 +17,7 @@ def rowCount(self, index=QModelIndex()):
         return len(self.addons)
 
     def data(self, index, role=Qt.DisplayRole):
-        if role == Qt.DisplayRole:
+        if role in Qt.DisplayRole:
```

Comment:
And thi leads you to errors like https://ci.appveyor.com/project/parkouss/mozregression/build/1.0.460#L279

You should check yourself the appveyor and travis-ci logs, we can discuss that on irc.